### PR TITLE
Adjust materials-importer script to newest materials version

### LINKED ISSUE #1971: Various outlined icons are missing
This icons are once added and are now missing. They should be added again.
Screenshot 2020-09-22 at 08 53 34

## Expected Behavior

Icons should be available.

## Current Behavior

Icons were deleted and can not be used by our customers.

## Steps to Reproduce

1. Go to storybook
2. Open icons
3. Search for icons given above

## Possible Solution

Add all outlined icons

### PR DESCRIPTION
**Pull request by 01231**
Fixes #1971

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [x] Modifications still work in IE.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.
- [x] Dependencies to other components still work.

### CODE REVIEW

**Comment by 01231** [comment]
At the moment, the designers evaluate if outlined icons are needed. In case they are not, the script just needs to be updated for general icons.

**Comment by 01231** [comment]
Outlined icons are not needed by now. (Designers opinion)

File: src/components/00-materials/material-importer.sh [hunk 1]
```diff
@@ -10,25 +10,34 @@
...
-    if [[ $x == *"/production/"* ]]; then
-        # echo "Copy file: $x"
-        OPTIMIZED_FILENAME=$(printf '%s\n' "${x//$STRING_TO_REMOVE_1/}")
-        OPTIMIZED_FILENAME=$(printf '%s\n' "${OPTIMIZED_FILENAME//$STRING_TO_REMOVE_2/}")
-        cp -f $x $OPTIMIZED_FILENAME
-        mv -v $OPTIMIZED_FILENAME $ICON_DIR
-        # cp -f $OPTIMIZED_FILENAME $ICON_DIR
-        #   mv -v $x $ICON_DIR
-    fi
+     # echo "Copy file: $x"
```
Comment (raphaellueckl) [inline src/components/00-materials/material-importer.sh]: ☝️😳

File: src/components/00-materials/material-importer.sh [hunk 2]
```diff
@@ -10,25 +10,34 @@
...
+
+      if [[ $PARENT_FOLDER_NAME == "materialicons" ]]; then
+            OPTIMIZED_FILENAME=$(printf '%s\n' "${x//$STRING_TO_REMOVE/$ICON_NAME}")
+      fi
+
+      # uncomment below if outlined icons are needed
+      # if [[ $PARENT_FOLDER_NAME == "materialiconsoutlined" ]]; then
+      #       ICON_NAME_OUTLINED="${ICON_NAME}_outlined"
+      #       OPTIMIZED_FILENAME=$(printf '%s\n' "${x//$STRING_TO_REMOVE/$ICON_NAME_OUTLINED}")
+      # fi
```
Comment (raphaellueckl) [inline src/components/00-materials/material-importer.sh:36]: So currently we have no outlined icons?

File: src/components/00-materials/material-importer.sh [hunk 3]
```diff
@@ -10,25 +10,34 @@
...
+
+     cp -f $x $OPTIMIZED_FILENAME
```
Comment (raphaellueckl) [inline src/components/00-materials/material-importer.sh:38]: Why is this necessary? You copy the file and move the copied file?

File: src/components/00-materials/material-importer.sh [hunk 4]
```diff
@@ -10,25 +10,34 @@
...
+     mv -v $OPTIMIZED_FILENAME $ICON_DIR
+     # cp -f $OPTIMIZED_FILENAME $ICON_DIR
```
Comment (raphaellueckl) [inline src/components/00-materials/material-importer.sh]: unnecessary line 41 and 42

**Review by raphaellueckl — CHANGES_REQUESTED** [review]
nice, you're stepping up your bash scripting game 😎

File: src/components/00-materials/material-importer.sh (hunk 2 above)
Comment (01231) [reply src/components/00-materials/material-importer.sh:36]: I mean the designer decided so, but if we need them this snipped would add them.

File: src/components/00-materials/material-importer.sh (hunk 1 above)
Comment (01231) [reply src/components/00-materials/material-importer.sh]: Was already in the code from you xD
Should I delete it?

File: src/components/00-materials/material-importer.sh (hunk 3 above)
Comment (01231) [reply src/components/00-materials/material-importer.sh:38]: I haven't touched this code. Thought there is a specific reason...

File: src/components/00-materials/material-importer.sh (hunk 4 above)
Comment (01231) [reply src/components/00-materials/material-importer.sh]: Dito

File: src/components/00-materials/material-importer.sh (hunk 1 above)
Comment (01231) [reply src/components/00-materials/material-importer.sh]: done

File: src/components/00-materials/material-importer.sh (hunk 4 above)
Comment (01231) [reply src/components/00-materials/material-importer.sh]: removed

**Review by raphaellueckl — APPROVED** [review]

**Pull request merged**